src/components/navigation/Footer.astro

```diff
@@ -60,9 +94,42 @@ const offices = [
     & p {
       font-size: 1.125rem;
       font-weight: 400;
-      color: var(--grey-900);
     }
   }
+
+  .footer__red {
+    background: linear-gradient(180deg, #ffc2be 0%, #ef826e 45%, #e45738 100%);
+
+    p {
+      color: var(--text-primary);
+    }
+
+    a {
+      color: var(--grey-800);
+    }
+  }
+
+  .footer__black {
+    background: var(--background-dark);
+
+    p {
+      color: var(--text-primary-on-dark);
+    }
+
+    a {
+      color: var(--grey-50);
+    }
+  }
+
+  .footer__curly {
+    height: 565px;
+    object-fit: contain;
+    position: absolute;
+    right: 0;
+    margin-right: -16px;
```

Comment:
Kanskje fjerne margin right. Så ser det bedre ut på store skjermer. Evt bruke container/media queries for å sjekke om den er max width så fjerne margin. (Kan også heller sette right -x px. Kan også være lurt å lage en padding variabel slik at padding og posisjonen stemmer over ens om man endrer padding. Feks når paddingen endrer seg fra desktop til mobil
<img width="709" alt="image" src="https://github.com/user-attachments/assets/612b5ec4-baf4-48a5-9ff7-aabef4649d5c" />